tests/Tests/200__monitor_and_manage/200__metrics/201__billing_metrics.robot

```diff
@@ -17,15 +20,62 @@ ${METRIC_RHODS_UNDEFINED}           cluster:usage:consumption:rhods:undefined:se
 
 *** Test Cases ***
 Verify OpenShift Monitoring results are correct when running undefined queries
```

Comment:
could you write this as Camel Case?

Reply:
it applies aso to the other TCs in this suite

Reply:
done